design/decisions/why-license/index.qmd

```diff
@@ -0,0 +1,67 @@
+
+The **Permissive licenses** gives you all the above mentioned rights, but they do not enforce that derived works or new works created by taking bits of source code from the original product are made available under the same license. These type of licenses are generally seen as more friendly to commercial enterprises as they will allow companies to use bits of code in proprietary software without having to release the source code for free. Examples of Permissive licenses are [Apache](https://www.apache.org/licenses/LICENSE-2.0), BSD (no matter the number of clauses, e.g., [BSD 2-Clause](https://opensource.org/license/bsd-2-clause/) or [BSD 3-Clause](https://opensource.org/license/bsd-3-clause/)), and [MIT](https://opensource.org/license/mit/).
+
+#### CLA versus DCO
+
+Some open source projects are asking contributors to sign up to a Contributor License Agreement (CLA) or, alternatively, a Developer Certificate of Origin (DCO).
+
+Looking at a number of **CLA**s (in particular [Threema](https://threema.ch/en/open-source/cla), [Meta](https://code.facebook.com/cla), and [ImageWorks](https://www.imageworks.com/technology/opensource/cla)), it seems that these projects are mainly concerned with the following:
+
+- A person submitting code that they are not the copyright holder for.
+- Withdrawal of the right to use the submitted code.
+- The need to apply for patents in future for parts or all of the source code.
+- A change in the type of license that the source code was given at the time of contribution.
+
+There are also some agreements that touch on the subject of loss and damages which may arise from the use of a particular section of code, as well as how a request for support will be dealt with in future.
+
+An alternative to a CLA is a **DCO**. The **DCO** was first employed by the Linux Foundation in 2004 and is basically a short document that confirms that the person contributing code is allowed to do so, gives permission for the project to subsequently use it, by adding a Signed-off-by line to their commit message (for an example see the [BeeWare projects DCO](https://beeware.org/contributing/how/dco/what/)).
+
+## Decision Outcome
+
+
+It is in our stated [goals](../../software-architecture/introduction.qmd) that the Seedcase software is available forcommercial enterprises as well as academic/healthcare organizations and groups. Aligning with our stated [Guiding Principles](../software-architecture/introduction.qmd#guiding-principles), we will use a **permissive** license as it will be the best fit for the Seedcase project. Of the permissive license types we will go with the MIT License.
```

Comment:
"forcommercial" --> "for commercial". 
Add why ther permissive license *MIT* and not e.g., Apache or BSD?